api/comms/websocket.go

```diff
@@ -0,0 +1,268 @@
+package comms
+
+import (
+	"encoding/json"
+	"sync"
+	"time"
+
+	"bridgerton.audius.co/trashid"
+	"github.com/gofiber/contrib/websocket"
+	"go.uber.org/zap"
+)
+
+const (
+	sendQueueSize      = 128 // per-connection limit
+	pingInterval       = 30 * time.Second
+	readIdleTimeout    = 60 * time.Second
```

Comment:
this seems long but i'm not too familiar. why 1 min?

Reply:
Actually, turns out this is controlling how long we will wait for an incoming ping message before considering the connection bad. Since I'm not sure how often clients send pings (or if they do at all), I'm going to do some investigating and either set this to zero (no timeout), or leave it as it is.

Reply:
Okay follow up:
Read conection deadline is overall time we will wait to receive the next message from the client. The only messages the clients will send are responses to pings. We send pings every 30s to detect connection health. So this would mean we are willing to wait up to 30 seconds for a 'pong' response. I can set it a little lower. But I don't want to go much lower than it is right now lest we start dropping healthy connections.